test/app1.1/index.js

```diff
@@ -0,0 +1,6 @@
+var binary = require('node-pre-gyp');
+var path = require('path')
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var binding = require(binding_path);
```

Comment:
Please end the file with one and only one \n to get rid of the ⛔ below.  Repeat in other files.

Reply:
Not going to happen… I don’t work here. Sorry.